projects/addon-commerce/components/input-card-grouped/input-card-grouped.template.html

```diff
@@ -26,7 +26,7 @@
                 tuiInputMode="numeric"
                 class="t-input t-input_card"
                 [class.t-input_inert]="cardPrefilled"
-                [class.t-input_hidden]="isCardCollapsed"
+                [class.t-input_empty]="card.length < 1 && focused"
```

Comment:
nit: how do you like `!card.length && focused`  ?

Reply:
fixed